buildkite/scripts/build_tarball.sh

```
@@ -9,6 +9,8 @@ echo "$PRIVATE_PEM_B64" | base64 --decode > /tmp/private.pem
 echo "$PUBLIC_PEM_B64" | base64 --decode > /tmp/public.pem
 export PRIVATE_KEY_PASSPHRASE="$(echo $PRIVATE_KEY_PASSPHRASE_B64 | base64 --decode)"
 export BACALHAU_RELEASE_TOKEN=$(buildkite-agent secret get BACALHAU_RELEASE_TOKEN)
+export DOCKER_USERNAME=$(buildkite-agent secret get DOCKER_USERNAME)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)